kotlin/src/main/kotlin/net/jqwik/kotlin/api/BuildersExtensions.kt

```diff
@@ -9,10 +9,8 @@ import java.util.function.BiFunction
  * Convenience function for Kotlin to not use backticked `in` function.
  */
 @API(status = API.Status.EXPERIMENTAL, since = "1.6.0")
-fun <B, T> BuilderCombinator<B>.use(arbitrary: Arbitrary<T>, combinator: (B, T) -> B): BuilderCombinator<B>
+fun <B, T: Any?> BuilderCombinator<B>.use(arbitrary: Arbitrary<T>, combinator: (B, T) -> B): BuilderCombinator<B>
     where B : Any {
-    // This explicit conversion form combinator to toFunction is necessary
```

Comment:
The comment is still valid, isn't it?

Reply:
It is not. `toFunction` is no longer neded

Reply:
It's still being used in your code, though. I'm confused.

Reply:
I am afraid it is not used. Please provide an exact link where `toFunction` is used in `BuilderExtensions.kt`

Reply:
Line 15 in BuilderExtensions.kt reads 
`return this.use(arbitrary).`in`(toFunction)`

Reply:
Please ensure you use the latest code.
Here's my branch: https://github.com/vlsi/jqwik/blob/nullness_annotations/kotlin/src/main/kotlin/net/jqwik/kotlin/api/BuildersExtensions.kt

Line 15 is the final closing curly brace: https://github.com/vlsi/jqwik/blob/abe7b1e99f6656eb26cdeea864f90f03867e4313/kotlin/src/main/kotlin/net/jqwik/kotlin/api/BuildersExtensions.kt#L15

Reply:
Now I see the final code. Maybe GH cached the old state or I viewed it before you pushed. Whatever.